src/ios/VLCPlugin.m

```diff
@@ -730,7 +852,8 @@ - (void) vlc_setNowPlayingInfo:(NSDictionary*)info retrieveLockscreenArt:(BOOL)r
             MPMediaItemPropertyArtist: artist,
             MPNowPlayingInfoPropertyElapsedPlaybackTime: elapsedPlaybackTime,
             MPMediaItemPropertyPlaybackDuration: playbackDuration,
-            MPMediaItemPropertyArtwork: mediaItemArtwork
+            MPMediaItemPropertyArtwork: mediaItemArtwork,
+            MPNowPlayingInfoPropertyPlaybackRate : playbackRate,
```

Comment:
Why is the MPNowPlayingInfoPropertyPlaybackRate added? It's fine, just wondering for my own knowledge.